### DISCUSSION TITLE
Creator kit dependency on Dataverse Legacy

### DISCUSSION THREAD

**Discussion by dineshary** [microsoft/powercat-creator-kit#281] [General]
I got a DLP issue when the app using creator kit components is moved to another environment. 

**Business connector: shared_sharepointonline cannot be used with non-business connector: /providers/Microsoft.PowerApps/apis/shared_commondataservice.**

So, i have to make the Microsoft Dataverse Legacy connector as Business connector in the DLP policy of the target environment. Why is there a dependency on Dataverse legacy? I hear that dataverse legacy is getting depreciated very soon. What should be done if that happens?

**Comment by denise-msft** [comment]
@dineshary which components are you using? Were you using any components copied over from the model driven app reference solution, specifically from the reference app?
